pom.xml

```diff
@@ -1776,6 +1776,41 @@
         <findbugs.excludeFilterFile>src/main/findbugs/exclude-filter.xml</findbugs.excludeFilterFile>
       </properties>
     </profile>
+    <profile>
+      <id>overrides</id>
```

Comment:
Although added to check for overrides, the profile activates errorprone compiler, and should probably be named `use-errorprone` or `compile-with-errorprone` or similar.

Also, it might be worth adding a brief XML comment at the top of the profile (above the `<id>`) to briefly explain what the profile is for.

Reply:
Any profile you can put in a POM, can also be placed in settings.xml. You could just cut/paste from pom.xml to settings.xml and it will work the same way, activated by `-Pname` or whatever activation is configured for that profile. Additionally, you can automatically activate profiles (regardless of whether they are in pom.xml or settings.xml) by setting the `<activeProfiles>` section appropriately: https://maven.apache.org/settings.html#Active_Profiles

Reply:
Evidently, I'm wrong. Some elements, including `<build>` cannot appear in the settings.xml profiles.